modules/Library/library_manage_catalog_addProcess.php

```diff
@@ -23,7 +23,14 @@
 
 include './moduleFunctions.php';
 
-$URL = $session->get('absoluteURL').'/index.php?q=/modules/'.getModuleName($_POST['address']).'/library_manage_catalog_add.php&name='.$_GET['name'].'&gibbonLibraryTypeID='.$_GET['gibbonLibraryTypeID'].'&gibbonSpaceID='.$_GET['gibbonSpaceID'].'&status='.$_GET['status'].'&gibbonPersonIDOwnership='.$_GET['gibbonPersonIDOwnership'].'&typeSpecificFields='.$_GET['typeSpecificFields'];
+$address = $_POST['address'] ?? '';
+$name = $_GET['name'] ?? '';
+$gibbonLibraryTypeID = $_GET['gibbonLibraryTypeID'] ?? '';
+$gibbonSpaceID = $_GET['gibbonSpaceID'] ?? '';
+$status = $_GET['status'] ?? '';
+$gibbonPersonIDOwnership = $_GET['gibbonPersonIDOwnership'] ?? '';
+$typeSpecificFields = $_GET['typeSpecificFields'] ?? '';
+$URL = $session->get('absoluteURL').'/index.php?q=/modules/'.getModuleName($address).'/library_manage_catalog_add.php&name='.$name.'&gibbonLibraryTypeID='.$gibbonLibraryTypeID.'&gibbonSpaceID='.$gibbonSpaceID.'&status='.$status.'&gibbonPersonIDOwnership='.$gibbonPersonIDOwnership.'&typeSpecificFields='.$typeSpecificFields;
```

Comment:
For consistency, this could remove the string concatenation in between the variables.